### DISCUSSION TITLE
Getting error from hoomd import group

### DISCUSSION THREAD

**Discussion by Dipanjali-1993** [glotzerlab/hoomd-blue#1731] [Support] [answered]
Hi all,

I have recently started using HOOMD-Blue for my simulations. I have installed  "hoomd=4.5.0=*cpu*" in the conda environemnt using the latest version of miniconda. I can successfully run the  example script given in the link http://glotzerlab.engin.umich.edu/hoomd-blue/. 

But when I am using 
from hoomd import group

Its throwing me an error 
ImportError: cannot import name 'group' from 'hoomd' (/home/sen/miniconda3/lib/python3.12/site-packages/hoomd/__init__.py)

Same is true for using from hoomd import deprecated

Can you please help

**Comment by joaander** [comment, marked as answer]
`hoomd.group` and `hoomd.deprecated` do not exist. I indicate this by the fact that they do not appear in [the documentation](https://hoomd-blue.readthedocs.io/en/v4.5.0/index.html).

The [migration guide](https://hoomd-blue.readthedocs.io/en/v4.5.0/migrating.html) may help you migrate from an earlier version.
